.codecov.yml

```diff
@@ -0,0 +1,14 @@
+coverage:
+  status:
+    project:
+      default:
+        # Commits pushed to master should not make the overall
+        # project coverage decrease by more than 1%:
+        target: auto
+        threshold: 1%
```

Comment:
as noted in other comment, can we hold a hard line on this and disallow any reduction in coverage?

Also, purely out of curiosity: is there a way to bypass the check (with committer approval) in an emergency situation?

Reply:
Nope I think you need admin to override the blockers, which none of the committers get. You'd have to open an apache INFRA ticket :(

Reply:
It just occurred to me that _this file_ provides a perfectly suitable (and auditable!) mechanism for emergency exemptions... the committer could just add one or more ignore paths with a comment in the file documenting the need, so we would have a written record of anything that accidentally gets left in there